ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/RequestHelper.java

```diff
@@ -68,14 +68,14 @@ public static String getAmazonS3InitParams(final boolean isBucketRequired, final
         if (!isBucketRequired) {
             return "";
         } else if (isObjectRequired) {
-            return "const char* bucket_name, const char* object_name";
+            return "const char *const bucket_name, const char *const object_name";
         }
-        return "const char* bucket_name";
+        return "const char *const bucket_name";
```

Comment:
https://matt.sh/sytycc

the first const refers to the pointer, the 2nd to the value it points to.